### PR TITLE
docs(readme) improve about hoop.dev and add backed by area

### PR DESCRIPTION
**Pull request by mtmr0x**
I've improved a bit our description and added a "Backed By" area at the end of our Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,19 @@
...
+
+ * Passwordless Auth, No Certificates: older gateways require high-maintenance certificate authorities. Hoop uses OIDC and Oauth2 for authentication, letting your IDP handle everything behind the scenes. Forget about certificates!
+ * Open-source SSO: support for Okta, Keycloak, Jumpcloud, and others. There is no need for Enterprise versions to integrate your own IDP. You're not limited to GitHub sign-in.
+ * Session recording: Linux, Docker, Kubernetes, Mysql, Postgres, MongoDB, and many more.
+ *	Just-in-time access: grant time-bound sessions using groups synced from your IDP.
+ * Slack and Teams Access Requests: Chatbot approval workflows can be done without leaving your chat app.
+
+Discover the unique capabilities that only Hoop can offer. From packet manipulation to web and proxy modes, Hoop is designed to meet your diverse needs.
+
+* Manipulate packets: Programmatically changes the gateway's environment and each connection's packets in real-time. Check out the Secrets Manager iration  e xample.
```
Comment (rogefm) [inline README.md]: "Check out the Secrets Manager iration e xample."
I'm not sure, but is it migration example?

```suggestion
* Manipulate packets: Programmatically changes the gateway's environment and each connection's packets in real-time. Check out the Secrets Manager migration example.
```

File: README.md [hunk 2]
```diff
@@ -48,3 +60,11 @@ Hoop.dev is an access gateway for databases and servers. Our AI-powered automati
...
 | Frankfurt (eu-central-1) | [![Launch Stack](https://cdn.rawgit.com/buildkite/cloudformation-launch-stack-button-svg/master/launch-stack.svg)](https://eu-central-1.console.aws.amazon.com/cloudformation/home?region=eu-central-1#/stacks/quickcreate?templateURL=https%3A%2F%2Fhoopdev-platform-cf-eu-central-1.s3.eu-central-1.amazonaws.com%2Flatest%2Fhoopdev-platform.template.yaml) |
 | Sydney (ap-southeast-2) | [![Launch Stack](https://cdn.rawgit.com/buildkite/cloudformation-launch-stack-button-svg/master/launch-stack.svg)](https://ap-southeast-2.console.aws.amazon.com/cloudformation/home?region=ap-southeast-2#/stacks/quickcreate?templateURL=https%3A%2F%2Fhoopdev-platform-cf-ap-southeast-2.s3.ap-southeast-2.amazonaws.com%2Flatest%2Fhoopdev-platform.template.yaml) |
+
+## Development
+
+Check out our [Development Documentation](/DEV.md)
+
+## Backed by
+
+![Backed By YC, Valor, GFC, Quiet and L2 Ventures](backed-by.png)
```
Comment (rogefm) [inline README.md]: The image is broke.

**Review by rogefm — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mtmr0x) [reply README.md]: thanks, some problems with markdown + Grammarly 😂

File: README.md [hunk 3]
```diff
@@ -26,7 +26,19 @@
...
+ * Open-source SSO: support for Okta, Keycloak, Jumpcloud, and others. There is no need for Enterprise versions to integrate your own IDP. You're not limited to GitHub sign-in.
+ * Session recording: Linux, Docker, Kubernetes, Mysql, Postgres, MongoDB, and many more.
+ *	Just-in-time access: grant time-bound sessions using groups synced from your IDP.
+ * Slack and Teams Access Requests: Chatbot approval workflows can be done without leaving your chat app.
+
+Discover the unique capabilities that only Hoop can offer. From packet manipulation to web and proxy modes, Hoop is designed to meet your diverse needs.
+
+* Manipulate packets: Programmatically changes the gateway's environment and each connection's packets in real-time. Check out the [Secrets Manager integration example](https://hoop.dev/docs/learn/secrets-manager).
+ * Web and proxy modes: Existing gateways lock you into either a web client interface or a proxy that requires desktop agents. Hoop gives you both options.
+ * Custom connections: bring your own CLI or hide complex options from developers. Becomes for users.
```
Comment (sandromello) [inline README.md]: "Becomes for users"  - What do you mean by that? It's a typo?

Comment (mtmr0x) [reply README.md]: yes, thanks!

**Review by rogefm — APPROVED** [review]

**Review by sandromello — APPROVED** [review]

**Pull request merged**